.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -2270,10 +2279,12 @@ stages:
     timeoutInMinutes: 60 #default value
     strategy:
       matrix:
-        centos7:
+        x64:
           baseImage: centos7
+          artifactSuffix: linux-x64
         alpine:
```

Comment:
shouldn't you rename alpine into `musl-x64` ?

Reply:
> shouldn't you rename alpine into musl-x64 

We could, yes. But my inclination was to leave it as alpine for now, as that's a "friendlier" name, and is harder to confuse. This is purely for use in the UI, so I don't think it's a big deal personally...

> I do not want to confuse, but shouldn't be linux-glibc-x64 and linux-musl-x64 ?
because alpine is x64 linux disribution, but with its own libc.

I'm inclined to say no, because 

- we use `linux-x64` and linux-musl-x64` everywhere in our directories, so it's consistent with that
- Microsoft use `linux-x64` and linux-musl-x64` everywhere too